### ISSUE TITLE
Use tooltip for elm-format messages

### ISSUE THREAD

**Issue by pravdomil** [klazuka/intellij-elm#588]
the ballon at the right bottom corner is sticky and do not goes away, so I would copy the prettier behaviour

**elm-format**
<img width="399" alt="image" src="https://user-images.githubusercontent.com/2387356/72225366-0fc96200-3585-11ea-9b8d-9c6f5030eafa.png">

**prettier**
<img width="290" alt="image" src="https://user-images.githubusercontent.com/2387356/72225339-e01a5a00-3584-11ea-82ee-28b157ff3ca0.png">

**Comment by klazuka** [comment]
Agreed. This would be better. Or at a minimum I should make the notification disappear automatically.
